definitions/shells.yaml

```diff
@@ -0,0 +1,106 @@
+name: RootUserShellConfigs
+doc: Linux root shell configuration files.
+collectors:
+- collector_type: FILE
+  args:
+    path_list:
+      - '/root/.bashrc'
+      - '/root/.bash_profile'
+      - '/root/.bash_logout'
+      - '/root/.cshrc'
+      - '/root/.ksh'
+      - '/root/.logout'
+      - '/root/.profile'
+      - '/root/.tcsh'
+      - '/root/.zlogin'
+      - '/root/.zlogout'
+      - '/root/.zprofile'
+labels: [Configuration Files]
+supported_os: [Linux]
+---
+name: UsersShellConfigs
+doc: Linux root shell configuration files.
+collectors:
+- collector_type: FILE
+  args:
+    path_list:
+      - '%%users.homedir%%/.bashrc'
+      - '%%users.homedir%%/.bash_profile'
+      - '%%users.homedir%%/.bash_logout'
+      - '%%users.homedir%%/.cshrc'
+      - '%%users.homedir%%/.ksh'
+      - '%%users.homedir%%/.logout'
+      - '%%users.homedir%%/.profile'
+      - '%%users.homedir%%/.tcsh'
+      - '%%users.homedir%%/.zlogin'
+      - '%%users.homedir%%/.zlogout'
+      - '%%users.homedir%%/.zprofile'
+labels: [Configuration Files]
+supported_os: [Linux, Darwin]
+---
+name: RootUserShellHistory
+doc: Linux root shell configuration files.
+collectors:
+- collector_type: FILE
+  args:
+    path_list:
+      - '/root/.bash_history'
+      - '/root/.sh_history'
+      - '/root/.zhistory'
+      - '/root/.zsh_history'
+labels: [Configuration Files]
+supported_os: [Linux]
+---
+name: UsersShellHistory
+doc: Linux root shell configuration files.
+collectors:
+- collector_type: FILE
+  args:
+    path_list:
+      - '/%%users.homedir%%/.bash_history'
+      - '/%%users.homedir%%/.sh_history'
+      - '/%%users.homedir%%/.zhistory'
+      - '/%%users.homedir%%/.zsh_history'
+labels: [Configuration Files]
+supported_os: [Linux, Darwin]
+---
+name: GlobalShellConfigs
+doc: Linux global shell configuration files.
+collectors:
+- collector_type: FILE
+  args:
+    path_list:
+      - '/etc/bashrc'
+      - '/etc/bash.bashrc'
+      - '/etc/kshrc'
+      - '/etc/csh.cshrc'
+      - '/etc/csh.login'
+      - '/etc/csh.logout'
+      - '/etc/profile'
+      - '/etc/zsh/zlogin'
+      - '/etc/zsh/zlogout'
+      - '/etc/zsh/zprofile'
+      - '/etc/zsh/zshenv'
+      - '/etc/zsh/zshrc'
+      - '/etc/zshenv'
+      - '/etc/zshrc'      
+labels: [Configuration Files]
+supported_os: [Linux, Darwin]
+---
+name: SudoersConfiguration
```

Comment:
these need to be removed from linux.yaml.  I feel like they should be in a separate file that covers the commonalities between os x and linux.  Maybe nix.yaml?